### PR TITLE
Added data for Uduak and Gerald

### PR DESCRIPTION
**Pull request by BravoClassic**
## Data files for various sections

- [ ] Add education, experience, and projects data for page content.

### CODE REVIEW

**Comment by BravoClassic** [comment]
@acushlakoncept review the pull request I have made.

File: _data/education2.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- course: Software Developer 
+  institute: Microverse
+  dates: 2020 - 2021
+  logo: uob.png
+
+- course: B.Eng Agricultural Engineering, Major in Farm Mechanization
+  institute: University of Uyo
+  dates: 2005 - 2010
+  logo: uob.png
```
Comment (acushlakoncept) [inline _data/education2.yml]: Let's remove this one, and allow only the Software Developer only

File: _data/education2.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+- course: Software Developer 
+  institute: Microverse
+  dates: 2020 - 2021
+  logo: uob.png
```
Comment (acushlakoncept) [inline _data/education2.yml]: You could use this image for the school logo
microverse

File: _data/experience2.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+- role: Senoir Code Reviewer
+  company: Microverse
+  dates: May 2021 - Present
+  logo: fellowship.svg
+
```
Comment (acushlakoncept) [inline _data/experience2.yml]: You can still use the Microverse logo here
Microverse

File: _data/experience2.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+- role: VKS Nigeria Construction Ltd
+  company: IT Chief
+  dates: April 2020 - Present
+  logo: fellowship.svg
+
+- role: Network Engineer
+  company: Akwa Ibom eLibrary
+  dates: Jun 2012 - Jul 2016
+  logo: fellowship.svg
```
Comment (acushlakoncept) [inline _data/experience2.yml]: We can take this off, since two experience would be enough

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
Hi @BravoClassic, 

Good job so far!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: _data/education2.yml (hunk 1 above)
Comment (BravoClassic) [reply _data/education2.yml]: Okay!

**Comment by BravoClassic** [comment]
Hi @acushlakoncept,
Made all the required changes.
Let me know if it is okay.
Happy coding! 👏👏

**Review by acushlakoncept — APPROVED** [review]
Thank you for making the updates 👍 

Please feel free to pick up another issue

**Pull request merged**